stablehlo/reference/Ops.cpp

```diff
@@ -81,5 +81,13 @@ Tensor eval(TanhOp op, const Tensor &operand) {
   return result;
 }
 
+Tensor eval(SubtractOp op, const Tensor &lhs, const Tensor &rhs) {
+  Tensor result(op.getType());
+  for (auto i = 0; i < lhs.getNumElements(); ++i) {
+    result.set(i, lhs.get(i) - rhs.get(i));
+  }
+  return result;
+}
```

Comment:
Looks like this will need to be rebased on top of HEAD.

Reply:
Unrelated to this PR, but it looks like cosine and tanh needs to be fixed as well. Will make PR addressing accumulated fixes after this one.